tests/EdgeToEdge/Routes/ShowDonationConfirmationRouteTest.php

```diff
@@ -83,18 +93,18 @@ private function newBookedAnonymousPayPalDonation( FunFunFactory $factory ): Don
 		return $donation;
 	}
 
-	private function assertDonationDataInResponse( Donation $donation, string $responseContent ): void {
+	private function assertDonationDataInResponse( Donation $donation, Payment $payment, string $responseContent ): void {
 		$donor = $donation->getDonor();
 		$personName = $donor->getName();
 		$personNameValues = $personName->toArray();
 		$physicalAddress = $donor->getPhysicalAddress();
-		/** @var DirectDebitPayment $paymentMethod */
-		$paymentMethod = $donation->getPaymentMethod();
+
+		$paymentData = $payment->getDisplayValues();
 
 		$this->assertStringContainsString( 'donation.id: ' . $donation->getId(), $responseContent );
-		$this->assertStringContainsString( 'donation.amount: ' . $donation->getAmount()->getEuroString(), $responseContent );
-		$this->assertStringContainsString( 'donation.interval: ' . $donation->getPaymentIntervalInMonths(), $responseContent );
-		$this->assertStringContainsString( 'donation.paymentType: ' . $donation->getPaymentMethodId(), $responseContent );
+		$this->assertStringContainsString( 'donation.amount: ' . $paymentData['amount'], $responseContent );
+		$this->assertStringContainsString( 'donation.interval: ' . $payment->getInterval()->value, $responseContent );
```

Comment:
I don't understand why you did not use `$paymentData['interval']` here, it'd be more constistent